resources/assets/less/bem/beatmap-scoreboard-table.less

```diff
@@ -127,8 +127,14 @@
     }
 
     &:hover {
+      color: #fff;
+
       > td {
-        background-color: #ccc;
+        background-color: #555;
+
+        > a {
```

Comment:
should be inside `__user-link`

Reply:
Moving this to the `&__user-link:hover` block means that link colour will always be grey, even when the row is hovered, until the user moves their cursor over the link itself.

Reply:
Inside `.@{_top}__body-row:hover &`.